filtering/flink_table_api_java/README.md

```diff
@@ -55,7 +55,7 @@ EnvironmentSettings envSettings = ConfluentSettings.fromResource("/cloud.propert
 TableEnvironment tableEnv = TableEnvironment.create(envSettings);
 ```
 
-Let's filter one of Confluent Cloud's built-in example tables. These reside in the read-only `marketplace` database of the `examples` catalog. The source code in this example uses The Table API's [`Table.filter`](https://docs.confluent.io/cloud/current/flink/reference/functions/table-api-functions.html#table-interface-sql-equivalents) method to find orders greater than or equal to 50 (we also could have used the equivalent [`Table.where`](https://docs.confluent.io/cloud/current/flink/reference/functions/table-api-functions.html#table-interface-sql-equivalents) method):
+Let's filter one of Confluent Cloud's built-in example tables. These reside in the read-only `marketplace` database of the `examples` catalog. The source code in this example uses the Table API's [`Table.filter`](https://docs.confluent.io/cloud/current/flink/reference/functions/table-api-functions.html#table-interface-sql-equivalents) method to find orders greater than or equal to 50 (we also could have used the equivalent [`Table.where`](https://docs.confluent.io/cloud/current/flink/reference/functions/table-api-functions.html#table-interface-sql-equivalents) method):
```

Comment:
how about a more active voice instead of 👍 

> These reside in the read-only `marketplace` database of the `examples` catalog

.
simething like:

>  " You can find these tables in the .."